stage1/init/kvm/network.go

```diff
@@ -158,7 +158,9 @@ func GenerateNetworkInterfaceUnits(unitsPath string, netDescriptions []netDescri
 			return errwrap.Wrap(fmt.Errorf("failed to create network unit file %q", unitName), err)
 		}
 
-		rlog.Printf("network unit created: %q in %q (iface=%q, addr=%q)", unitName, unitsPath, ifName, address)
+		if debug {
```

Comment:
I apologize, I was testing the wrong stage1 image. Also, the debug flag is (at least, was) meant to give more information when set to true, not be silent when false. That would be the job of a `--quiet` flag.

However, I think the better way to do what this is trying is to use `rlog.SetOutput(ioutil.Discard)` at the entry point of the package. This way you don't need to special case each debug output.